workspaces/repo-tools/.changeset/proud-ducks-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/cli': patch
+---
+
+fix(cli): New workspaces ignores now private repositories when release plugins
```

Comment:
Not needed, we don't release this package to NPM currently.

Reply:
Sure.  I've removed the file and rebased the branch. :crossed_fingers: